Prüfschritte/de/11.1.4.4 Text auf 200% vergrößerbar.adoc

```diff
@@ -42,17 +42,9 @@ Muss nur geprüft werden, wenn die Systemfunktion zum vergrößern nicht vorhand
 
 === 3. Hinweise
 
-Werden Apps getestet, kommen die Funktionen Zoomen und / oder Nur-Text-Vergrößerung des Betriebssystems zum Einsatz:
+* Die Zoomfunktion (Vergrößerung des gesamten Bildausschnitts) des Betriebssystems wird in der Regel eine 200%ige Vergrößerung ermöglichen. Funktioniert die Zoomfunktion mit der zu testenden Software, ist dieser Prüfschritt bestanden. Wir empfehlen jedoch dringend, dass native Apps die häufig genutzte dynamische Textvergrößerng untertützen, da die Bedürfnisse von sehbehinderten Nutzenden durch die Zoom-Funktion der Plattform nicht vollständig abgedeckt werden. Probleme mit der Nur-Text-Vergrößerung sollten zumindest angemerkt werden.
```

Comment:
Typos: Textvergrößerung unterstützen

(Vergrößerung des gesamten Bildausschnitts)
DF: Der Bildausschnitt *verkleinert* sich in der Vergrößerung, die Formulierung ist unglücklich.

DF: Was unterstützt werden sollte, ist ja nicht eigentlich die Textvergrößerungsfunktion, sondern der Einsatz von dynamic fonts, auf die sich die Textvergrößerungsfunktion dann tatsächlich auswirkt (da müsste man noch mal checken, wie die bei iOS und Android genau heißen).